terratorch/cli_tools.py

```diff
@@ -98,9 +98,32 @@ def save_prediction(prediction, input_file_name, out_dir, dtype:str="int16"):
     file_name = os.path.basename(input_file_name)
     file_name_no_ext = os.path.splitext(file_name)[0]
     out_file_name = file_name_no_ext + "_pred.tif"
-    logging.info(f"Saving output to {out_file_name} ...")
+    logger.info(f"Saving output to {out_file_name} ...")
     write_tiff(result, os.path.join(out_dir, out_file_name), metadata)
 
+def import_custom_modules(custom_modules_path:str=None) -> None:
+
+    if custom_modules_path:
+
+        custom_modules_path = Path(custom_modules_path)
+
+        if custom_modules_path.is_dir():
+
+            # Add 'custom_modules' folder to sys.path
+            workdir = custom_modules_path.parents[0]
+            module_dir = custom_modules_path.name
+
+            sys.path.append(workdir)
+
+            try:
+                importlib.import_module(module_dir)
+                logger.info(f"Found {custom_modules_path}")
+            except Exception:
+                raise Exception(f"It was not possible to import modules from {custom_modules_path}.")
+        else:
+            logger.info(f"The modules path {custom_modules_path} not found.")
```

Comment:
I don't think we want to just log this, I think we want to raise an error. For example:
`raise ValueError(f"Custom module path '{custom_modules_path}' not found")`